.travis.yml

```diff
@@ -13,7 +13,7 @@ python:
   - 3.5
 
 env:
-  - CONDA_PACKAGES="numpy scipy pytest future"
+  - CONDA_PACKAGES="numpy scipy pytest future jupyter ipython matplotlib nomkl scikit-learn numexpr"
```

Comment:
Why are jupyter, ipython, matplotlib, scikit-learn, and numexpr dependencies? (Yes, they are needed to run your notebook, but they are not needed for the package itself, or the tests.)